package.json

```diff
@@ -89,12 +89,12 @@
 		"prosemirror-tables-contently": "^1.1.2",
 		"prosemirror-transform": "^1.6.0",
 		"prosemirror-utils": "^1.0.0-0",
-		"prosemirror-view": "^1.26.4",
+		"prosemirror-view": "^1.26.5",
 		"proxy-polyfill": "^0.3.2",
 		"tippy.js": "^6.3.7",
-		"vue": "^2.6.14",
+		"vue": "^2.7.8",
```

Comment:
Let's maybe add vue-template-compiler as a dependency directly then dependabot can also take care of bumping them in sync

Reply:
Done.